omero/sysadmins/server-ldap.txt

```diff
@@ -139,7 +139,11 @@ the value of ``new_user_group``, which can have several different values:
   group is mapped via ``group_mapping``. You can prefix this value with
   ``filtered_`` to have the ``group_filter`` applied to the attribute values,
   i.e. ``:filtered_dn_attribute:memberOf`` will mean that only the values of
-  memberOf which match ``group_filter`` will be considered.
+  memberOf which match ``group_filter`` will be considered. Note that if an
+  attribute specified in ``omero.ldap.group_mapping`` does not constitute a
+  part of the Distinguished Name (DN) as determined by your LDAP server then it
+  can only be found by using ``:attribute:`` or ``:filtered_attribute:``
+  instead. Typical DN attributes are: DC, CN, OU, O, STREET, L, ST, C and UID.
```

Comment:
I will clarify that sentence.
On 5 Mar 2015 18:46, "Josh Moore" <notifications@github.com> wrote:

> In omero/sysadmins/server-ldap.txt
> <https://github.com/openmicroscopy/ome-documentation/pull/1132#discussion_r25889553>
> :
>
> > @@ -139,7 +139,11 @@ the value of ``new_user_group``, which can have several different values:
> >    group is mapped via ``group_mapping``. You can prefix this value with
> >    ``filtered_`` to have the ``group_filter`` applied to the attribute values,
> >    i.e. ``:filtered_dn_attribute:memberOf`` will mean that only the values of
> > -  memberOf which match ``group_filter`` will be considered.
> > +  memberOf which match ``group_filter`` will be considered. Note that if an
> > +  attribute specified in ``omero.ldap.group_mapping`` does not constitute a
> > +  part of the Distinguished Name (DN) as determined by your LDAP server then it
> > +  can only be found by using ``:attribute:`` or ``:filtered_attribute:``
> > +  instead. Typical DN attributes are: DC, CN, OU, O, STREET, L, ST, C and UID.
>
> I didn't immediately understand this sentence. You mean something like
> "DNs are typically composed of DC, CN, OU, ..."? Otherwise, no problems
> with the content.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openmicroscopy/ome-documentation/pull/1132/files#r25889553>
> .
>